lib/querystring.js

```diff
@@ -159,7 +159,7 @@ function stringify(obj, sep, eq, options) {
   sep = sep || '&';
   eq = eq || '=';
 
-  var encode = QueryString.escape;
```

Comment:
We can't make this change in case userland overrides the `QueryString.escape` function.